dwave/cloud/client.py

```diff
@@ -383,7 +383,7 @@ def from_config(cls, config_file=None, profile=None, client=None,
         return _clients[_client](**config)
 
     def __init__(self, endpoint=None, token=None, solver=None, proxy=None,
-                 permissive_ssl=False, **kwargs):
+                 permissive_ssl=False, timeout=60, **kwargs):
```

Comment:
Why 60?

Reply:
Just a reasonable default. Since users are not expecting a timeout, we don’t want to timeout too often. But it’s customizable for power users.